migrations/4_stablecoin_converter.js

```diff
@@ -1,7 +1,12 @@
-const argv = require("../src/migration_utilities")
 const migrateStablecoinConverter = require("../src/migration_scripts_stablecoinConverter/migrate_PoC_dfusion")
+const argv = require("yargs")
```

Comment:
It would have had to be 

```js
const { argv } = require("../migration_utilities")
```

Right now it was an object with three fields (argv, getDependency, isDevelopmentNetwork)